manage.py

```diff
@@ -139,6 +141,38 @@ def __getitem__(self, item):
     else:
         SESSION_ENGINE = "django.contrib.sessions.backends.db"
 
+    if not DJANGO_1_9:
```

Comment:
because the middleware list is the same, please store the list of middleware in a variable outside the conditional and just do the assignment inside the conditional.